### ISSUE TITLE
Prep miniDREAM website prior to course start (2022)

### ISSUE THREAD

**Issue by lopierra** [Sage-Bionetworks/minidream-r-env#20]
See details within issues from 2021.

- [x] #4 
- [x] #5 
- [x] #6 
- [x] #7 
- [x] #8 
- [x] #9 
- [ ] #13 
- [x] ~#14~
- [ ] Update Additional Resources section  - DO FOR NEXT YEAR
- [x] #18
- [x] ~Set up CompOnc Zoom meetings for lectures/office hours~ Not having office hours this year; in future ask Amber to set up to Zoom meetings

**Comment by lopierra** [comment]
Pierrette's janky method for creating a new Project using `synapseformation`:

1. Make the yml file using the template [here](https://github.com/Sage-Bionetworks/synapseformation/blob/main/templates/long.yaml). Make sure to use the exact structure with hyphens, indents, etc. I used RStudio as my text editor (New File > Text File > Save As > `minidream_2022_formation.yml`)
2. Open a terminal (I tried to use the RStudio terminal but it didn't work for some reason, so I used git bash)
3. Use `cd` to navigate to folder with the yml file
4. `python -m venv venv`
5. `source .venv/Scripts/activate` (this is for Windows - use `bin` instead of `Scripts` for Mac)
6. `pip install synapseformation`
7.  `synapse login -u "username" -p "password" --rememberMe`
8. `synapseformation create --template_path minidream_2022_formation.yml`
